### PR TITLE
ttt-exam:0.1.0

### PR DESCRIPTION
**Pull request by jomaway**
I am submitting
- [x] a new package

Description: Template for creating an exams using ttt-utils.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/ttt-exam/0.1.0/thumbnail.png
Comment (laurmaedje) [inline packages/preview/ttt-exam/0.1.0/thumbnail.png:1]: The thumbnail should show a single page of the template as-initialized.

File: packages/preview/ttt-exam/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[package]
+name = "ttt-exam"
+version = "0.1.0"
+compiler = "0.11.0"
+entrypoint = "lib/lib.typ"
+authors = ["Jomaway <https://github.com/jomaway>"]
+license = "MIT"
+description = "A collection of tools to make a teachers life easier (german)."
```
Comment (laurmaedje) [inline packages/preview/ttt-exam/0.1.0/typst.toml:8]: The description is possibly a copy-paste error from `ttf-utils`?

Comment (jomaway) [reply packages/preview/ttt-exam/0.1.0/typst.toml]: oh, you are right.

File: packages/preview/ttt-exam/0.1.0/thumbnail.png
Comment (jomaway) [reply packages/preview/ttt-exam/0.1.0/thumbnail.png:1]: ok, didn't know that.

Comment (jomaway) [reply packages/preview/ttt-exam/0.1.0/thumbnail.png:1]: The output depends if you use a --input solution=true or not so i thought it is a good thing to include both versions in the thumbnail

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**